frontend/assets/javascripts/src/modules/giraffe.js

```diff
@@ -16,7 +16,7 @@ define(['src/modules/form/validation/display','ajax'], function (display, ajax)
 
 
     function init() {
-        if (!document.querySelectorAll('.container-global--giraffe').length) {
+        if (!document.querySelectorAll('.container-global--giraffe .js-form').length) {
```

Comment:
and maybe we should just stick a more specific class on the giraffe contribute page